### PR TITLE
Disable update of ouput window while visual mode is active

### PR DESCRIPTION
**Pull request by ishitaku5522**
Fix https://github.com/lervag/vimtex/issues/1318

### CODE REVIEW

File: autoload/vimtex/compiler.vim [hunk 1]
```diff
@@ -174,6 +174,10 @@ function! vimtex#compiler#output() abort " {{{1
       return
     endif
 
+    if mode() ==# 'v' || mode() ==# 'V' || mode() ==# ''
```
Comment (lervag) [inline autoload/vimtex/compiler.vim]: Why are you also checking `mode() ==# ''` here?

Comment (lervag) [reply autoload/vimtex/compiler.vim]: Also, you can combine the first two by using `==?`, that is: `mode() ==? 'v'`.

**Comment by lervag** [comment]
Thanks! I'll merge this after we've settled my comments.

File: autoload/vimtex/compiler.vim (hunk 1 above)
Comment (ishitaku5522) [reply autoload/vimtex/compiler.vim]: Thank you for your review.

> Why are you also checking `mode() ==# ''` here?

On Github, it looks empty string but this is actually "^V" (input by Ctrl+V Ctrl+V) which means Ctrl+V for visual block mode.
You can see it in raw souce code:   
https://raw.githubusercontent.com/ishitaku5522/vimtex/no_output_update_on_visual/autoload/vimtex/compiler.vim

> Also, you can combine the first two by using `==?`, that is: `mode() ==? 'v'`.

I updated.

Comment (ishitaku5522) [reply autoload/vimtex/compiler.vim]: Is "\<C-v>" better for you?

Comment (lervag) [reply autoload/vimtex/compiler.vim]: Yes, I think `"\<c-v>` is better (I prefer to not capitalize the letters, though). Thanks!

**Comment by ishitaku5522** [comment]
Thank you for your waiting. I updated!

**Comment by lervag** [comment]
Thank you for noticing, reporting and fixing the issue! :)

**Pull request merged**